sys/Plugins/Transports/FtpTransport.php

```diff
@@ -131,9 +141,25 @@ public function get(): array {
 
 		//ftp_chdir($conn, $this->path);
 
-		$contents = ftp_nlist($conn, $this->path);
+		$contentsAll = ftp_mlsd($conn, $this->path);
+		//die(var_dump($contentsAll));
+
+		// Take only newest of all files which meet the filename pattern
+		$path = $this->path;
+		if ($path !== '' && substr($path, -1) !== '/') $path .= '/';
+		$contents = [];
+		$newest = '0';
```

Comment:
Let's make this 'newest' mechanic configurable, and I would think not the default behavior.

Reply:
I would also extract it into its own method, like
```suggestion
		if($this->newestFileOnly)
		$contents = $this->newestFile($contentsAll)
```
Then you also don't need $contentsAll

Reply:
OK, I made it configurable and extract the newest mechanic in its own function